runtime.go

```diff
@@ -1513,7 +1522,7 @@ func (r *Runtime) RunProgram(p *Program) (result Value, err error) {
 		vm.stash = &r.global.stash
 		vm.sb = vm.sp - 1
 	}
-	vm.prg = p
+	vm.setPrg(p)
```

Comment:
[q1] yes, I believe in some cases the ticks could be 0, but I believe that's fine. I'll let Calvin confirm that thought :) 

[q2] agreed. Now that we're closer to a solution we should start adding some tests. We don't need to check exact ticks, but we should make sure that the metrics contain the correct functions and that the ticks of an expensive JS function are higher than the ticks of trivial functions. It might also be worth testing that executing a function X times generates X times more ticks than executing the same function once.